charts/rancher-logging/0.1.5/questions.yml

```diff
@@ -1 +1 @@
-rancher_min_version: 2.3.4-rc1
\ No newline at end of file
+rancher_min_version: 2.4.0-rc1
```

Comment:
Do we really need the min version for 2.4.0?

Reply:
I'd like to keep the Rancher min version the same and add in comments into the `app-readme.md` of what's the changelog.